packages/components/src/components/tab-group/tab-group.custom.styles.ts

```diff
@@ -5,8 +5,9 @@ export default css`
   /* Write custom CSS here */
   :host {
     --track-width: var(--syn-border-width-small);
-    --track-color: var(--syn-color-neutral-300);
+    --track-color: var(--syn-panel-border-color);
     --indicator-width: var(--syn-border-width-x-large);
+    --indicator-color: var(--syn-interactive-emphasis-color, --syn-color-primary-600);
```

Comment:
The `var()` is missing around the fallback color




```suggestion
    --indicator-color: var(--syn-interactive-emphasis-color, var(--syn-color-primary-600));
```